Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ISISIndirectDiffractionReduction.py

```diff
@@ -189,20 +195,46 @@ def PyExec(self):
 
             # Process workspaces
             for ws_name in workspaces:
+                monitor_ws_name = ws_name + '_mon'
+
                 # Subtract empty container if there is one
                 if self._container_workspace is not None:
                     Minus(LHSWorkspace=ws_name,
                           RHSWorkspace=self._container_workspace,
                           OutputWorkspace=ws_name)
 
                 if self._vanadium_ws:
+                    van_ws = mtd[self._vanadium_ws[index]]
```

Comment:
Can I suggest that we extract a variable, `van_ws_name = self._vanadium_ws[index]` and use this below to avoid repeatedly looking up the value in the array?